### PR TITLE
Fix command restart 

### PR DESCRIPTION
**Pull request by vwalek**
Fix for bugzilla - https://bugzilla.redhat.com/show_bug.cgi?id=1640164

related to 3.9, 3.7, 3.6, 3.5 releases and less.

### CODE REVIEW

File: install_config/topics/sssd_for_ldap_failover_configure_openshift.adoc [hunk 1]
```diff
@@ -42,13 +42,12 @@ need to adjust these steps to fit your cluster.
...
 
-. Start {product-title} with the updated configuration:
+. Restart {product-title} with the updated configuration:
 +
 ----
-# openshift start \
-    --public-master=https://openshift.example.com:8443 \
-    --master-config=/etc/origin/master/master-config.yaml \
-    --node-config=/etc/origin/node-node1.example.com/node-config.yaml
+systemctl restart atomic-openshift-master-api
```
Comment (kalexand-rh) [inline install_config/topics/sssd_for_ldap_failover_configure_openshift.adoc]: Same as the other PR, please add $ or #.

**Comment by kalexand-rh** [comment]
@jianlinliu, will you PTAL?

**Comment by kalexand-rh** [comment]
@stuartchuan, will you PTAL?

**Comment by stuartchuan** [comment]
The changes LGTM.

**Comment by kalexand-rh** [comment]
Thank you @stuartchuan!
 
@vwalek, please squash so I can merge.

**Comment by kalexand-rh** [comment]
Squashed here: https://github.com/openshift/openshift-docs/pull/14598
I'm going to merge that one.

**Pull request closed without merging**